chart/templates/configmap.yaml

```diff
@@ -14,6 +14,7 @@ data:
       metricsAddr: "127.0.0.1:8080"
       leaderElection: true
       metricsSink: {{ .Values.controller.metricsSink | quote }}
+      notifierSinks: {{ .Values.controller.notifierSinks }}
```

Comment:
Could you add a per-notifier configuration here?
I would expect a notifier, for instance Slack, to be configurable like:

```yaml
controller:
  notifiers:
    noop:
      enabled: false
    slack:
      enabled: true
      tokenFilePath: /etc/chaos-controller/slack_token
      context:
        clusterName: foo
```

or with common context values

```yaml
controller:
  notifiers:
    context:
      clusterName: foo
    noop:
      enabled: false
    slack:
      enabled: true
      tokenFilePath: /etc/chaos-controller/slack_token
```

Reply:
Done